aiohttp/web_response.py

```diff
@@ -122,7 +124,8 @@ def __init__(
         else:
             self._headers = CIMultiDict()
 
-        self.set_status(status, reason)
+        self._status = status
+        self._reason = reason or REASON_PHRASES.get(status, "")
```

Comment:
I don't see a reason for the duplication here?

Reply:
The idea was to avoid the `self.prepared` check since it was impossible to be an issue in `__init__`.  I'll break it out into an underscore function instead.